wasm2c/wasm-rt-impl.c

```diff
@@ -241,8 +311,15 @@ bool wasm_rt_is_initialized(void) {
 }
 
 void wasm_rt_free(void) {
+  if (!wasm_rt_is_initialized()) {
+    fprintf(stderr, "wasm_rt_free: wasm runtime not initialized.\n");
+    return;
+  }
```

Comment:
Maybe just `assert(wasm_rt_is_initialized() && ...` (no need for early return since its basically undefined behaviour to do this?)

Reply:
Done.